lib/Entity/Playlist.php

```diff
@@ -518,6 +518,10 @@ public function delete($options = [])
             }
         }
 
+        // region delete, the parentId is the regionPlaylist playlistId, we want to remove all linked subplaylists in that regionPlaylist
```

Comment:
can we expand this comment so that we say why it is safe to do this here - something like:

> we are deleting a region so any sub-playlists attached to it will be deleted - we want to remove all link records from the closure table using the parentId as that is the ID of the region specific playlist.

Reply:
comment suggested because it took me a long while to work out whether this was safe or not